.github/workflows/upload-plugin-s3.yml

```diff
@@ -0,0 +1,60 @@
+name: Upload latest plugin to S3
+
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+
+env:
+  AWS_PLUGINBUCKET_NAME: dss-plugins
+  AWS_PLUGINBUCKET_IAM_ROLE: arn:aws:iam::236706865914:role/dss-plugin-eks-clusters-github
```

Comment:
Two new variables have been created named DKU_PLUGINS_AWS_IAM_ROLE and DKU_PLUGINS_S3_BUCKET.